packages/ai/core/embed/embed-many.ts

```diff
@@ -73,6 +74,13 @@ Only applicable for HTTP-based providers.
   functionality that can be fully encapsulated in the provider.
   */
   providerOptions?: ProviderOptions;
+
+  /**
+   * Maximum number of concurrent requests.
+   *
+   * @default Infinity
+   */
+  concurrency?: number;
```

Comment:
rename to `maxParallelCalls` or `maxParallelRequests` or `maxConcurrentRequests`. If we rename to `Concurrent` here it would be good to do the same in the model spec

Reply:
yep that's a clearer name, updated